docs/docs/rest-api/public/api/api.raml

```diff
@@ -7,6 +7,7 @@ baseUri: ../
 
 uses:
   app: v2/types/app.raml
+  enrichedTask: v2/types/enrichedTask.raml
```

Comment:
IIRC the `EnrichedTask` format was named as such because it serialized more information that a task alone would have. I think it's OK to keep this name, but from an API perspective, it _could_ also just be named `task.raml`.